app/src/main/res/drawable/shape_round_corner.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android" >
```

Comment:
WARNING: [UnusedResources] The resource `R.drawable.shape_round_corner` appears to be unused
 Unused resources make applications larger and slow down builds.